nominees/synthea.json

```diff
@@ -10,25 +13,38 @@
   ],
   "SDGs": [
     {
-      "SDGNumber": 3
+      "SDGNumber": 3,
+      "evidenceText": "The “Good Health and Well-Being” goal highlights that a lack of data hinders understanding of the true impacts of disease. Synthea provides open-source software and freely available synthetic open data sets formatted to international standards. These resources lower the barrier of entry for innovators across the globe to build data-centric health solutions and digital health solutions. Using Synthea, health innovators can accelerate their work safely, securely, and publicly without accessing real data.",
+      "evidenceURL": "https://aspe.hhs.gov/synthetic-health-data-generation-engine-accelerate-patient-centered-outcomes-research"
+    },
+    {
+      "SDGNumber": 4,
+      "evidenceText": "The “Quality Education” goal highlights inclusive and equitable lifelong learning opportunities. Innovators also can leverage Synthea to strengthen their health information technology knowledge and skills for future jobs, research, and/or entrepreneurship opportunities to advance health and well-being. "
```

Comment:
I get where they are going, but is this really SDG 4? I'm not sure whether the basis of Synthea being an emerging technology project is sufficient validation of SDG 4.

The easiest solution would be to adapt this for SDG 9: `Build resilient infrastructure, promote inclusive and sustainable industrialization and foster innovation`. In particular, I am thinking of [target 9.b](https://sdgs.un.org/goals/goal9): `Support domestic technology development, research and innovation in developing countries, including by ensuring a conducive policy environment for, inter alia, industrial diversification and value addition to commodities`.

Reply:
Resolved in commit 4afa096